egs2/TEMPLATE/tts1/tts.sh

```diff
@@ -66,6 +65,11 @@ n_mels=80                  # The number of mel basis.
 f0min=80  # Maximum f0 for pitch extraction.
 f0max=400 # Minimum f0 for pitch extraction.
 
+# X-Vector related
+use_xvector=false          # Whether to use x-vector.
+xv_tool=sb  # Toolkit to use (Speechbrain: sb, ESPnet:esp, kaldi)
+xv_model=speechbrain/spkrec-ecapa-voxceleb  # kaldi: http://kaldi-asr.org/models/8/0008_sitw_v2_1a.tar.gz
```

Comment:
I do not like too short variable name.
Why don't you use 
- xv_tool -> xvector_tool
- xv_model -> xvector_model
- sb -> speechbrain
- esp -> espnet

Reply:
And I want to keep the default behavior since many recipe instruction follows kaldi version.